test/TestHelper.cpp

```diff
@@ -108,6 +108,12 @@ void ImportTest::importState(json_spirit::mObject& _o, State& _state)
 		BOOST_REQUIRE(o.count("storage") > 0);
 		BOOST_REQUIRE(o.count("code") > 0);
 
+		bigint biValue256 = bigint("115792089237316195423570985008687907853269984665640564039457584007913129639936");
+		if (bigint(o["balance"].get_str()) >= biValue256)
+			BOOST_THROW_EXCEPTION(Exception() << errinfo_comment("State 'balance' is equal or greater than 2**256") );
```

Comment:
Add a specific exception type such as "ValueTooLarge" or "Stackoverflow" to Excpetions.h, and use that instead of a general Exception. Same for exceptions below.